test/index/functions/methods.snapshot.cc

```diff
@@ -0,0 +1,166 @@
+  // Virtual methods and inheritance
+  
+  struct S0 {
+//       ^^ definition [..] S0#
+    virtual void v1() {
+//               ^^ definition [..] S0#v1(49f6e7a06ebc5aa8).
+      v2();
+//    ^^ reference [..] S0#v2(49f6e7a06ebc5aa8).
+    }
+    virtual void v2() {}
+//               ^^ definition [..] S0#v2(49f6e7a06ebc5aa8).
+    virtual void v3() {}
+//               ^^ definition [..] S0#v3(49f6e7a06ebc5aa8).
+    virtual void v4() {}
+//               ^^ definition [..] S0#v4(49f6e7a06ebc5aa8).
+    virtual void v5() = 0;
+//               ^^ reference [..] S0#v5(49f6e7a06ebc5aa8).
```

Comment:
This reference has no corresponding definition, because the check for forward declarations also excludes `= 0`. Maybe we need some special casing here, because such a method may have a doc comment (and it will be dropped by the code as it stands).